src/libraries/Microsoft.Bcl.AsyncInterfaces/ref/Microsoft.Bcl.AsyncInterfaces.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);netstandard2.1</TargetFrameworks>
-    <Nullable>disable</Nullable>
+    <Nullable>enable</Nullable>
```

Comment:
Was the reason this was like this in the first place because we only built for netstandard2.0 and netfx, both of which don't support nullable annotations?

Reply:
Either that, or we just never got to it / prioritized it.  It's so easy to enable, though, I just went ahead and did so.  It'd be nice to eventually get to zero src disables :)